docs: add client SDKs to README (#78)
openmeterio/openmeter: README.md

- [Metering Kubernetes Pod Execution Time](/examples/ingest-kubernetes-pod-time-go)
- [Usage Based Billing with Stripe](/examples/export-stripe-go)

## Client SDKs

Currently, we offer the following Client SDKs:

- [Node.js](/api/client/node)
- [Go](/api/client/go)

> For languages where an SDK isn't available yet, we encourage using the [OpenAPI definition](/api/openapi.yaml). It can be utilized with the [OpenAPI Generator](https://openapi-generator.tech/) to generate your own client.

Reviewer: ```suggestion
For languages where an SDK isn't available yet, we encourage using the [OpenAPI definition](/api/openapi.yaml). We recommend using the [OpenAPI Generator](https://openapi-generator.tech/) to generate your own client.
```
